music/src/core/player.ts

```diff
@@ -409,29 +399,35 @@ class DrumKit {
 export class Player extends BasePlayer {
   private drumKit = DrumKit.getInstance();
 
-  private bassSynth =
-      new Tone.Synth({volume: 5, oscillator: {type: 'triangle'}}).toMaster();
+  private bassSynth = new Tone.Synth({
+    volume: 5,
+    oscillator: { type: 'triangle' },
+  }).toDestination();
 
-  private polySynth = new Tone.PolySynth(10).toMaster();
+  private polySynth = new Tone.PolySynth().toDestination();
```

Comment:
why is the (10) argument not needed anymore?

Reply:
it's a new tonejs version change: `PolySynth does not require a polyphony value. Voice allocation and disposing is done automatically based on demand.`

Reply:
Got it, thanks!